.github/workflows/pypi.yml

```diff
@@ -0,0 +1,32 @@
+name: "PyPI publishing"
```

Comment:
License header?

Reply:
we don't have license header on other github action workflow files currently either:
- https://github.com/apache/thrift/blob/master/.github/workflows/build.yml
- https://github.com/apache/thrift/blob/master/.github/workflows/cmake.yml

since this is a cherry-pick of #2928 we already merged, if we deem it necessary to add license header to all 3 files I'd suggest we do them in separated PR (probably after 0.20.0 release), what do you think?

Reply:
created https://github.com/apache/thrift/pull/2933 and also added to this PR.